File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/AspNetUser.vb
Imports System.Collections.Generic

Public Class AspNetUser
    Public Shared Function Add(value As Models.AspNetUser) As Integer
        Return Data.AspNetUser.Add(value)
    End Function

    Public Shared Function Edit(value As Models.AspNetUser) As Integer
        Return Data.AspNetUser.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.AspNetUser.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.AspNetUser
        Return Data.AspNetUser.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.AspNetUser
        Return Data.AspNetUser.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.AspNetUser)
        Return Data.AspNetUser.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171036/Business/AspNetUser.cs
using System.Collections.Generic;

namespace Business
{
    public class AspNetUser
    {
	    private static Data.AspNetUser obj = new Data.AspNetUser();

        public static int Add(Models.AspNetUser value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.AspNetUser value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.AspNetUser GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.AspNetUser GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.AspNetUser> GetAll()
        {
            return obj.GetAll();
        }
    }
}